fpsim/locations/kenya/data/method_time_coefficients.csv

```diff
@@ -1,64 +1,55 @@
-meanlog,1.786422667,Other.mod,lognormal
-sdlog,0.422155552,Other.mod,lognormal
-"age_grp_fact(0,18]",-0.397191956,Other.mod,lognormal
-"age_grp_fact(18,20]",-1.786422667,Other.mod,lognormal
-"age_grp_fact(20,25]",0,Other.mod,lognormal
-"age_grp_fact(25,35]",1.162085828,Other.mod,lognormal
-"age_grp_fact(35,50]",2.282028383,Other.mod,lognormal
\ No newline at end of file
+"estimate","coef","se","method","functionform","country"
+2.11517824974575,"meanlog",0.0273003509383565,"None","lnorm","Kenya"
+1.29041965480421,"sdlog",0.0116652830677841,"None","lnorm","Kenya"
+-0.162425317334029,"age_grp_fact(0,18]",0.053511254441146,"None","lnorm","Kenya"
+0.0916533525560237,"age_grp_fact(18,20]",0.0536383510261167,"None","lnorm","Kenya"
+0.128964912019441,"age_grp_fact(25,35]",0.0367422179490106,"None","lnorm","Kenya"
+1.02158311122685,"age_grp_fact(35,50]",0.0500591744552768,"None","lnorm","Kenya"
+2.35946187878322,"meanlog",0.0759002018821134,"Pill","lnorm","Kenya"
+1.30328951224842,"sdlog",0.0325645381511583,"Pill","lnorm","Kenya"
+-0.627132100382602,"age_grp_fact(0,18]",0.184472778730867,"Pill","lnorm","Kenya"
+-0.571646219557213,"age_grp_fact(18,20]",0.167670426502694,"Pill","lnorm","Kenya"
+0.509506376666395,"age_grp_fact(25,35]",0.0947413189800796,"Pill","lnorm","Kenya"
+1.09898567733413,"age_grp_fact(35,50]",0.129134899503056,"Pill","lnorm","Kenya"
+1.61392883103592,"shape",0.16049792661208,"IUD","weibull","Kenya"
+43.5900002963957,"scale",6.49637743650689,"IUD","weibull","Kenya"
+-2.00913521915797,"age_grp_fact(0,18]",0.463946016248004,"IUD","weibull","Kenya"
+-0.360380859715765,"age_grp_fact(18,20]",0.342644694255669,"IUD","weibull","Kenya"
+0.457791445248868,"age_grp_fact(25,35]",0.182255102054402,"IUD","weibull","Kenya"
+1.2907944052172,"age_grp_fact(35,50]",0.293508072027006,"IUD","weibull","Kenya"
+2.96438266575112,"meanlog",0.0413253939956661,"Injectable","lnorm","Kenya"
+1.16974924403454,"sdlog",0.0204179909850332,"Injectable","lnorm","Kenya"
+-0.471387115806453,"age_grp_fact(0,18]",0.0992305253157196,"Injectable","lnorm","Kenya"
+-0.214652298416606,"age_grp_fact(18,20]",0.0885810873851957,"Injectable","lnorm","Kenya"
+0.435210558261227,"age_grp_fact(25,35]",0.0540455868990147,"Injectable","lnorm","Kenya"
+0.671318533985408,"age_grp_fact(35,50]",0.0788225996645395,"Injectable","lnorm","Kenya"
+2.96669981194057,"meanlog",0.171786872080675,"Condom","lnorm","Kenya"
+1.8458277252793,"sdlog",0.0801061575477104,"Condom","lnorm","Kenya"
+-0.944358020278272,"age_grp_fact(0,18]",0.239115412430351,"Condom","lnorm","Kenya"
+-0.295970986986498,"age_grp_fact(18,20]",0.271123262573453,"Condom","lnorm","Kenya"
+0.63769812541032,"age_grp_fact(25,35]",0.239867104685988,"Condom","lnorm","Kenya"
+1.04143926245061,"age_grp_fact(35,50]",0.309028443327074,"Condom","lnorm","Kenya"
+2.14701518071282,"shape",0.210521443743803,"Withdrawal","llogis","Kenya"
+12.2042876108707,"scale",2.04358981609571,"Withdrawal","llogis","Kenya"

```

Comment:
@mzimmermann-IDM can you please double check these parameters? The (shape, scale) pars for withdrawal were previously (0.76, 2.5) and are now (2.15, 12.2). This seems like a big change. A lot of the other parameters have changed too. The two figures below show the distributions on the methodtime branch (first fig) vs on this new branch (second fig).

Previously:
![duration_dists](https://github.com/user-attachments/assets/83640bd9-6be3-43bd-993a-2bddbf941b78)

On this branch:
![duration_dists_new](https://github.com/user-attachments/assets/44a4926e-8ff3-477d-bcf7-ba2ba2522703)

Reply:
@robynstuart ok solved this one, I had just saved in a different format and I didn't realize it wasn't transforming the coefficients before saving. Should be fixed with my most recent commit.